go.work

```diff
@@ -22,4 +22,5 @@ use (
 	./pkg/gofr/datasource/scylladb
 	./pkg/gofr/datasource/solr
 	./pkg/gofr/datasource/surrealdb
+	pkg/gofr/datasource/dbresolver
```

Comment:
I would have kept the same format and I would have ordered them alphabeticaly, as they were, no?

Also, it would decrease the risk of conflicts